automation/src/test/scala/org/broadinstitute/dsde/firecloud/pages/AuthenticatedPage.scala

```diff
@@ -31,6 +41,21 @@ abstract class AuthenticatedPage(implicit webDriver: WebDriver) extends FireClou
     def clickSignOut(): Unit = {
       click on (await enabled signOutLink)
     }
+
+    def filter(searchText: String) = {
+      await enabled filterInput
+      searchField(filterInput).value = searchText
+      pressKeys("\n")
+    }
+
+    def getUser() = {
+      await enabled accountDropdown
+      val userDropdownText = find(accountDropdown).get.text
+      emailRegEx findFirstIn userDropdownText match {
```

Comment:
It might be worthwhile to put a `span` with a `data-test-id` around the email address so we can read it out without the need for a regex.